_layouts/product.html

```diff
@@ -183,6 +183,18 @@
   <td {% if diff <= 0 %} class = "txt-linethrough" {% endif %} >      <!-- if the support finished add txt-linethrough class -->
     {% if releaseLink != "" %}
       <a href="{{releaseLink}}" title="Release Notes / Changelog">{{latestVersionNumber}}</a>
+{% if  diff > 1 %}
+      &ensp;
+      <a href="http://www.google.com/calendar/render?action=TEMPLATE
+&text=[endoflife.date] {{page.title}} {{r.releaseCycle}} EOL reached
+&dates={{r.eol | replace:'-',''}}T000000/{{r.eol | replace:'-',''}}T000000
+&details=It will be end of support for  {{page.title}} {{r.releaseCycle}}
+&trp=false
+&sprop=
+&sprop=name:" target="_blank" rel="nofollow">
+<img alt="Add to Calendar" src="/assets/calendar-with-plus-sign.svg" width="20" height="20">
```

Comment:
It may be prettier if the icon were vertically aligned with the version text. Or maybe if the icon were in its own column ?

Reply:
having a new column will make it looks ugly because it will have empty columns for eoled or versions without eol dates

Reply:
Comment superseded by https://github.com/endoflife-date/endoflife.date/pull/1937#pullrequestreview-1197600383.